### ISSUE TITLE
Failure in setting up dev environment

### ISSUE THREAD

**Issue by sayak119** [internetarchive/internet-archive-voice-apps#469]
Due to lack of Dashbot and Bespoken keys, developers face issues while deploying the action even though the keys aren't necessary.

**Error** : `TypeError: Cannot read property 'key' of undefined`
